docs/internals/testing.md

```diff
@@ -36,37 +36,39 @@ Documentation for (roughly) code in the `src/testing` directory.
- 1   \  .       74V 147/147C  85:148Jo  0Jd  0Jf  85:148Wo  122695Gf
- 4   /     .    74V 148/148C  86:149Jo  0Jd  0Jf  86:149Wo  122695Gf   2/4Pp  0/3Pq
- 1   \  .       74V 148/148C  85:148Jo  0Jd  0Jf  85:148Wo  122695Gf
- 2   \   .      74V 148/148C  85:148Jo  0Jd  0Jf  85:148Wo  122695Gf
- 0   \ .        74V 148/148C  87:150Jo  1Jd  0Jf  86:149Wo  122695Gf
- 3   \    .     74V 148/148C  85:148Jo  0Jd  0Jf  85:148Wo  122695Gf
- 2 $ #   #
- 4   /     .    74V 149/149C  87:150Jo  0Jd  0Jf  87:150Wo  122695Gf   2/4Pp  0/3Pq
- 0   \ .        74V 149/149C  87:150Jo  0Jd  0Jf  87:150Wo  122695Gf
- 3   \    .     74V 149/149C  86:149Jo  0Jd  0Jf  86:149Wo  122695Gf
- 1   \  .       74V 149/149C  86:149Jo  0Jd  0Jf  86:149Wo  122695Gf
- 4   /     .    74V 150/150C  87:150Jo  0Jd  0Jf  87:150Wo  122695Gf   1/4Pp  0/3Pq
- 0   \ .        74V 150/150C  87:150Jo  0Jd  0Jf  87:150Wo  122695Gf
- 1   \  .       74V 150/150C  87:150Jo  0Jd  0Jf  87:150Wo  122695Gf
- 3   \    .     74V 150/150C  87:150Jo  0Jd  0Jf  87:150Wo  122695Gf
- 4   /     .    74V 151/151C  91:154Jo  0Jd  0Jf  91:154Wo  122695Gf   4/4Pp  0/3Pq
+11   |            .     1V 126/126C  64:127Jo  0/_1J!  64:127Wo   <0:__0>   44857Gf  0G!   0G?
+ 0   \ .                1V 126/126C  64:127Jo  0/_2J!  63:126Wo   <0:__0>   44857Gf  0G!   0G?
+ 6   |       .          1V 126/126C  64:127Jo  0/_2J!  62:125Wo   <0:__0>   44857Gf  0G!   0G?
+ 7   |        .         1V 126/126C  64:127Jo  0/_2J!  64:127Wo   <0:__0>   44857Gf  0G!   0G?
+ 9   |          .       1V 126/126C  64:127Jo  0/_2J!  62:125Wo   <0:__0>   44857Gf  0G!   0G?
+ 1   /  .               1V 127/127C  64:127Jo  0/_0J!  64:127Wo   <0:__0>   44857Gf  0G!   0G?   1/4Pp  0/3Rq
+ 2   \   .              1V 127/127C  64:127Jo  0/_0J!  64:127Wo   <0:__0>   44857Gf  0G!   0G?
+ 9   |          .       1V 127/127C  64:127Jo  0/_2J!  63:126Wo   <0:__0>   44857Gf  0G!   0G?
+ 7   |        .         1V 127/127C  64:127Jo  0/_2J!  64:127Wo   <0:__0>   44857Gf  0G!   0G?
+ 6   |       .          1V 127/127C  64:127Jo  0/_2J!  63:126Wo   <0:__0>   44857Gf  0G!   0G?
+11   |            .     1V 127/127C  64:127Jo  0/_0J!  64:127Wo   <0:__0>   44857Gf  0G!   0G?
+10 ^ |           h      1V   0/_11C  15:_15Jo 31/31J!   0:_15Wo   <0:__0>   45056Gf  0G!   0G?
+ 0   \ .                1V 127/127C  64:127Jo  0/_1J!  63:126Wo   <0:__0>   44857Gf  0G!   0G?
+ 4   \     .            1V 127/127C  64:127Jo  0/_1J!  63:126Wo   <0:__0>   44857Gf  0G!   0G?
+ 5   \      .           1V 127/127C  64:127Jo  0/_0J!  64:127Wo   <0:__0>   44857Gf  0G!   0G?
+ 3   \    .             1V 127/127C  64:127Jo  0/_1J!  63:126Wo   <0:__0>   44857Gf  0G!   0G?
+ 8   |         .        1V 127/127C  64:127Jo  0/_1J!  63:126Wo   <0:__0>   44857Gf  0G!   0G?
+10 < ~           h      1V   0/_11C  15:_15Jo 31/31J!   0:_15Wo   <0:__0>   45056Gf  0G!   0G?
+10 > |           h      1V 119/119C  15:_15Jo 31/31J!   0:_15Wo   <0:123>   44877Gf  0G!   0G?
```

Comment:
Using `<min:max>` for the sync min/max since `<` and `>` denote the start and end of superblock sync in the "event" column.